Python/UpeachyaKansakar/tasks/task1/show_result.py

```diff
@@ -0,0 +1,46 @@
+import os
+import csv
+import argparse
+import operator
+import datetime as dt
+
+def calculatePercentage (score, total):
```

Comment:
```suggestion
def calculate_percentage (score, total):
```
Please follow snake case of the function and variable name

Reply:
:ok: